src/fsharp/FSharp.Core.Unittests/FSharp.Core/PrimTypes.fs

```diff
@@ -104,7 +104,7 @@ type LanguagePrimitivesModule() =
         
         // reference type
         let resultRef = LanguagePrimitives.GenericComparison "ABC" "ABCDE"
-        Assert.AreEqual(resultRef,-68)
+        Assert.AreEqual(sign resultRef,-1)
```

Comment:
let shouldEqual (expected:'a) (actual:'a) = Assert.AreEqual(expexted,actual)

Is used in many projects and also enforces the correct type on compile time